cmd/manager/main.go

```diff
@@ -25,23 +25,28 @@ import (
 	v1 "k8s.io/api/core/v1"
 	"k8s.io/apimachinery/pkg/util/intstr"
 	"sigs.k8s.io/controller-runtime/pkg/client/config"
-	logf "sigs.k8s.io/controller-runtime/pkg/log"
+	log "github.com/sirupsen/logrus"
```

Comment:
Is this version approved to be used? Looks like someone's individual github repo?

Reply:
`github.com/sirupsen/logrus`? that is what almost all other packages in `kfctl`is using.